dotnetv3/cross-services/README.md

```diff
@@ -0,0 +1,16 @@
+# Cross-service examples
+
+The examples in this folder use multiple Amazon Web Services to perform more
+complex scenarios.
+
+## Examples
+
+### [AnniversarySNS](./Anniversary_SNS)
+
+**Services used:** Lambda, DynamoDB, SNS
```

Comment:
I'd suggest putting full services titles in the 'Services used' section, and second use versions subsequently i.e.
AWS Lambda, Amazon DynamoDB, Amazon Simple Notification Service (Amazon SNS)